docker-compose.override.yml

```diff
@@ -7,6 +7,18 @@ services:
       - '443:443'
     volumes:
       - ./conf/ssl:/ssl
+      - ./conf/nginx/conf.d/dev.conf:/etc/nginx/conf.d/oscar.template
+
+  backup:
+    build: .
+    image: countable/s3-backup-job:1.0
+    volumes:
+      - $HOME/.aws/credentials:/root/.aws/credentials
```

Comment:
Can you document the step for setting up credentials and everything else for backups, in the README?

Reply:
added